tests/ecosystem/deployment/test_acm.py

```diff
@@ -1,14 +1,15 @@
 from ocs_ci.ocs.acm.acm import import_clusters_with_acm
 from ocs_ci.ocs.acm.acm import AcmAddClusters
-
+from ocs_ci.framework.testlib import acm_import
 
 ####################################################################################################
 # This file is placeholder for calling import ACM as test, until full solution will be implimented #
 ####################################################################################################
 
 
-def test_import_acm(setup_acm_ui):
-    acm_obj = AcmAddClusters(setup_acm_ui)
+@acm_import
+def test_acm_import(setup_acm_ui):
+    AcmAddClusters(setup_acm_ui)
     import_clusters_with_acm()
```

Comment:
This looks little bit strange - because there is instantiated the class `AcmAddClusters` here, but didn't stored in any variable and then there is the call of `import_clusters_with_acm()` function, which instantiated the class `AcmAddClusters` again and uses it.

Reply:
Yes, this was a forgotten line to delete since some of its methods used to be called here. We only need the `import_clusters_with_acm()` with the instance created there. Since it assigns the selenium driver to the obj.